Sources/Sentry/SentryFileManager.m

```diff
@@ -71,7 +77,8 @@ - (void)deleteAllFolders
 
 - (NSString *)uniqueAcendingJsonName
 {
-    return [NSString stringWithFormat:@"%f-%lu-%@.json", [[NSDate date] timeIntervalSince1970],
+    return [NSString stringWithFormat:@"%f-%lu-%@.json",
```

Comment:
I understand this is a refactor to use the date provider but a comment describing this template would come in handy.

Reply:
Very good point.

Reply:
Perfect! thanks